specs/activity-api.yaml

```diff
@@ -383,6 +385,11 @@ paths:
               schema:
                 $ref: 'common.yaml#/schemas/ErrorMessage'
 components:
+  securitySchemes:
+    app_key:
+      type: apiKey
+      in: header
+      name: Authorization
```

Comment:
we use https://swagger.io/docs/specification/authentication/bearer-authentication/, aren't we?

Reply:
Yes we are using that one.
There was a bug in the python client generator that required the above patch.
The bug has been fixed, so I updated this PR to use the schema as you linked.